play-ebean/src/main/java/play/db/ebean/DefaultEbeanConfig.java

```diff
@@ -73,6 +73,7 @@ public EbeanConfig parse() {
                 ServerConfig serverConfig = new ServerConfig();
                 serverConfig.setName(key);
                 serverConfig.loadFromProperties();
+                serverConfig.setH2ProductionMode(true);  // Since Ebean 9.1.1: Don't override Evolution
```

Comment:
Yes cause Ebean développer tel to Ebean to generate and apply automatically DDL when H2 is used. Because he start from the premise that H2 is only used on development mode. But Ebean internal DDL generation enter in conflict with Evolution used on Play project.

You can give a try. If you remove this line, unit tests will fail because Evolution try to create previously created DB tables.